Makefile

```diff
@@ -9,3 +9,20 @@ test-integration:
 
 test-sanity:
 	ansible-test sanity --docker -v --color --python $(PYTHON_VERSION) $(?TEST_ARGS)
+
+generate-api-client: build-client move-client-files
+
+build-client:
+	npx @openapitools/openapi-generator-cli generate \
+	-g python \
+	-i ./api/v1alpha1/openapi.yml \
+	-o ./tmp-client \
+	--additional-properties=generateSourceCodeOnly=true \
+	--global-property=apiDocs=false,modelDocs=false,apiTests=false,modelTests=false \
+	-c ./openapiconfig.json
+
+# TODO add explanation for this shenanigans
+move-client-files:
```

Comment:
Can we generate the files directly to the target folder? Why breaking this into 2 steps?

Reply:
This has been modified, the generated files have been moved to a different location then imported in a different way